pyne/alara.py

```diff
@@ -198,6 +205,110 @@ def photon_source_to_hdf5(filename, nucs='all', chunkshape=(10000,)):
     f.close()
 
 
+def response_to_hdf5(filename, response, chunkshape=(10000,)):
+    """Converts a plaintext output.txt file to an HDF5 version for
+    quick later use.
+
+    This function produces a single HDF5 file named <filename>.h5 containing the
+    table headings:
+
+        idx : int
+            The volume element index assuming the volume elements appear in xyz
+            order (z changing fastest) within the photon source file in the case of
+            a structured mesh or mesh.mesh_iterate() order for an unstructured mesh.
+        nuc : str
+            The nuclide name as it appears in the output file.
+        time : str
+            The decay time as it appears in the output file.
+        response : float
+            Supported response:
+                - decay_heat [W/cm3]
+                - specific_activity [Bq/cm3]
+                - alpha_heat [W/cm3]
+                - beta_heat [W/cm3]
+                - gamma_heat [W/cm3]
+                - wdr
+                - photon_source
+
+    Parameters
+    ----------
+    filename : str
+        The path to the file output.txt
+    response : str
+        Key word of the response
+    chunkshape : tuple of int
+        A 1D tuple of the HDF5 chunkshape.
+    """
+    f = open(filename, 'r')
+    f.seek(0)
+
+    dt = np.dtype([
+        ('idx', np.int64),
```

Comment:
maybe this should also be `mesh_idx`

Reply:
@gonuke About this part, I think we need to keep it as `idx`. The `idx` can be used in both voxel (`idx == mesh_idx`) and sub-voxel R2S (`idx != mesh_idx`), and also the activation responses (`idx == mesh_idx`).

As we are going to merge them together, it's better to keep this `idx`.